diff --git a/TestingProjForTechTalk/Constants.swift b/TestingProjForTechTalk/Constants.swift
index 145638d..c84205d 100644
--- a/TestingProjForTechTalk/Constants.swift
+++ b/TestingProjForTechTalk/Constants.swift
@@ -30,6 +30,7 @@ struct Constants {
         return UIDevice.current.name.addingPercentEncoding(withAllowedCharacters: CharacterSet.urlHostAllowed) ?? ""
     }
     func sendInfoParams() -> String{
-        return "?version=\(getVersion())&client=\(kClient)&deviceIdentifier=\(deviceModel)=\(kManufacturer)&deviceName=\(getDeviceName())&osVersion=\(getOSVersion())"
+        return "infoParamsRandom"
     }
 }
+//?version=\(getVersion())&client=\(kClient)&deviceIdentifier=\(deviceModel)=\(kManufacturer)&deviceName=\(getDeviceName())&osVersion=\(getOSVersion())
diff --git a/TestingProjForTechTalk/ViewController.swift b/TestingProjForTechTalk/ViewController.swift
index c9e1db1..b6d3248 100644
--- a/TestingProjForTechTalk/ViewController.swift
+++ b/TestingProjForTechTalk/ViewController.swift
@@ -16,5 +16,10 @@ class ViewController: UIViewController {
     }
 
 
+    
+    func thisIsATestFunction() {
+        print("I'm in a different file and will be used later for testing while applying a patch")
+    }
+    
 }
 
